src/backend/common/queries/database_query.py

```diff
@@ -29,3 +30,15 @@ def fetch_async(self) -> TypedFuture[QueryReturn]:
             # Type-hinting the tasklet decorator is hard, but it consumes
             # the generator and returns the overall future
             return query_result  # pyre-ignore
+
+    def fetch_dict(self, version: int) -> dict:
```

Comment:
I'm not sure I understand the `dict` return type bit - I thought `Dict` was what we used with types right?

Reply:
Like I'm sure this is a type somehow I just don't see where 😅